src/javascript/app_2/components/elements/data_table.jsx

```diff
@@ -166,40 +36,61 @@ class DataTable extends React.Component {
     }
 
     renderBodyRows() {
-        return this.state.display_data.map((transaction, id) => this.renderRow(transaction, id));
+        return this.props.data_source
+            .map((transaction, id) => this.renderRow(transaction, id));
     }
 
     renderHeaders() {
-        return this.props.columns.map(col => <th key={col.data_index}>{col.title}</th>);
+        return this.props.columns.map(col => <th className={col.data_index} key={col.data_index}>{col.title}</th>);
     }
 
-    render() {
-        const { pagination } = this.props;
+    renderTableClone() {
+        {/*
+            cloned table with one row for fixed header
```

Comment:
Not sure cloning with only the first row will cover all cases.
i.e. when there is a longer cell in another row that causes changing the columns width.

Reply:
@binary-hamedanchi I have applied `table-layout: fixed` to the table. 🙂
So, according to spec:
> The table and column widths are set by the widths of table and col or by the width of the first row of cells. Cells in other rows do not affect column widths.